student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'Person'
+
+class Student < Person
+  def initialize(age:, classroom:, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] The `:` is not required in the constructor parameters, kindly make sure to remove them

```suggestion
  def initialize(age, classroom, name: 'Unknown', parent_permission: true)
```